Claude to fix cpp

@claude
❌ Build cpp Image
Failed Step: Build and push cpp image

Error Details
2025-10-04T03:03:09.3572324Z #11 0.423 error: externally-managed-environment
2025-10-04T03:03:09.4304570Z #11 ERROR: process "/bin/sh -c pip3 install conan" did not complete successfully: exit code: 1
2025-10-04T03:03:09.4355580Z ERROR: failed to build: failed to solve: process "/bin/sh -c pip3 install conan" did not complete successfully: exit code: 1
2025-10-04T03:03:09.4827077Z ##[error]buildx failed with: ERROR: failed to build: failed to

--- a/docker-images/cpp/Dockerfile
+++ b/docker-images/cpp/Dockerfile
@@ -44,7 +44,7 @@ RUN git clone https://github.com/Microsoft/vcpkg.git /opt/vcpkg && \
     ln -s /opt/vcpkg/vcpkg /usr/local/bin/vcpkg
 
 # Install Conan package manager
-RUN pip3 install conan
+RUN pip3 install --break-system-packages conan
 
 # Install VS Code C++ extensions
 ENV VSCODE_EXTENSIONS="ms-vscode.cpptools,ms-vscode.cpptools-extension-pack,ms-vscode.cmake-tools,twxs.cmake,jeff-hykin.better-cpp-syntax"

